test/pgo_trace_SUITE.erl

```diff
@@ -13,57 +11,62 @@ all() ->
 
 init_per_suite(Config) ->
     application:ensure_all_started(pgo),
-
+    application:ensure_all_started(tls_certificate_check),
     {ok, _} = pgo_sup:start_child(default, #{pool_size => 1,
                                              port => 5432,
                                              database => "test",
                                              user => "test",
                                              password => "password",
                                              trace => true}),
-
+    _ = application:load(opentelemetry),
+    application:set_env(opentelemetry, processors, [{otel_simple_processor, #{}}]),
+    {ok, _} = application:ensure_all_started(opentelemetry),
     Config.
 
 end_per_suite(_Config) ->
     application:stop(pgo),
+    application:stop(opentelemetry),
+    application:unload(opentelemetry),
     ok.
 
 init_per_testcase(_, Config) ->
-    _ = application:load(opentelemetry),
-    application:set_env(opentelemetry, processors, [{otel_batch_processor, #{scheduled_delay_ms => 1}}]),
-    {ok, _} = application:ensure_all_started(opentelemetry),
-    otel_batch_processor:set_exporter(otel_exporter_pid, self()),
+    otel_simple_processor:set_exporter(otel_exporter_pid, self()),
     Config.
 
 end_per_testcase(_, _Config) ->
-    _ = application:stop(opentelemetry),
+    otel_simple_processor:set_exporter(none),
     ok.
 
-trace_query(_Config) ->
-    ?assertMatch(#{rows := [{empty}]},
-                 pgo:query("select '[1,1)'::int4range", [], #{})),
```

Comment:
int4range causes some wild flakiness with tracing on. In the basic test suite, they all appear to work. For now, I replace these with `'1'::int` to test the tracing bits.